src/PrestaShopBundle/Model/Product/AdminModelAdapter.php

```diff
@@ -589,6 +589,7 @@ private function getCustomFields()
 
         foreach ($customizationFields as $customizationField) {
             $baseObject = [
+                'id_customization_field' => $customizationField[$this->locales[0]['id_lang']]['id_customization_field'],
```

Comment:
I tried to fix this the same way months ago, and it was working well until I've discovered a lot of new notices in my logs, but maybe was cause of another bug. If you confirm there is no more notices, I'm happy :)